lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -275,6 +275,21 @@ def get_cpu_facts(self, collected_facts=None):
                 cpu_facts['processor_vcpus'] = (cpu_facts['processor_threads_per_core'] *
                                                 cpu_facts['processor_count'] * cpu_facts['processor_cores'])
 
+                if sys.version_info[0] >= 3:
```

Comment:
Import `PY3` from `six`
```suggestion
                if PY3:
```

Reply:
see below